examples/file-sharing/src/network.rs

```diff
@@ -60,13 +57,12 @@ pub(crate) async fn new(
         transport,
         ComposedBehaviour {
             kademlia: Kademlia::new(peer_id, MemoryStore::new(peer_id)),
-            request_response: request_response::Behaviour::with_codec(
-                FileExchangeCodec(),
-                iter::once((
+            request_response: Behaviour::<FileRequest, FileResponse>::new(
```

Comment:
I'd like to encourage users to reference these types by module so I think we should do that in our examples:

```suggestion
            request_response: request_response::cbor::Behaviour::<FileRequest, FileResponse>::new(
```

Reply:
@thomaseizinger Of course, I like this approach too, I initially thought it was a bit too many words, but that's ok